### ISSUE TITLE
How to install this plugin

### ISSUE THREAD

**Issue by doutori** [prettier-solidity/prettier-plugin-solidity#3]
Is this library publish to npm?
Do I need directly install from git repository, like this?
```
npm install git+https://github.com/mattiaerre/prettier-plugin-solidity.git
```

**Comment by mattiaerre** [comment]
Hello @doutori no this lib is not published on npm yet as it is still a WIP.
You can try that for sure.
I hope I can get back at this and publish a beta version pretty soon.
Happy to have some help if you like.
Thanks,

**Comment by mattiaerre** [comment]
closed thanks to #34
